Loop/Managers/LoopDataManager.swift

```diff
@@ -288,7 +288,6 @@ final class LoopDataManager: LoopSettingsAlerterDelegate {
     fileprivate var predictedGlucose: [PredictedGlucoseValue]? {
         didSet {
             recommendedAutomaticDose = nil
-            recommendedManualBolus = nil
```

Comment:
Removing this, as manual boluses are calculated on demand via `.recommendBolus()`